.travis.yml

```diff
@@ -6,9 +6,11 @@ script: ~/qubes-builder/scripts/travis-build
 # keep the log under 4MB limit
 env:
  - DIST_DOM0=fc25 USE_QUBES_REPO_VERSION=4.0 USE_QUBES_REPO_TESTING=1 VERBOSE=0
+ - DIST_DOM0=fc29 USE_QUBES_REPO_VERSION=4.0 USE_QUBES_REPO_TESTING=1 VERBOSE=0
```

Comment:
Dom0 and VM builds are the same, it's only waste of time to build both of them here.